packages/attune/src/server/repo/list.rs

```diff
@@ -26,15 +30,21 @@ pub struct ListRepositoryResponse {
 pub async fn handler(
     State(state): State<ServerState>,
     tenant_id: TenantID,
+    Json(req): Json<ListRepositoryRequest>,
 ) -> Result<Json<ListRepositoryResponse>, ErrorResponse> {
+    // TODO: In the managed cloud version of this CLI, we should hide the S3
+    // bucket and prefix fields because they're irrelevant.
     let repositories = sqlx::query!(
         r#"
-        SELECT id, name
+        SELECT id, name, s3_bucket, s3_prefix
         FROM debian_repository
-        WHERE tenant_id = $1
+        WHERE
+            tenant_id = $1
+            AND name LIKE '%' || $2 || '%'
         ORDER BY created_at ASC
         "#,
         tenant_id.0,
+        req.name.unwrap_or_default(),
     )
```

Comment:
[optional] this is probably fine for now (especially given the relatively low number of rows- hundreds to thousands should be no huge deal), but in the past i've seen this sort of action blow up psql's ability to properly plan the query. 

Consider using [`sqlx_conditional_queries`](https://docs.rs/sqlx-conditional-queries/latest/sqlx_conditional_queries/) or just filtering client side.